application/lib/templates.dart

```diff
@@ -123,7 +123,7 @@ class TemplateLibrary extends StatelessWidget {
             ),
           ],
           icon: Icon(Icons.cloud),
-        ),
+        ),/*
```

Comment:
why are we commenting out this code?

Reply:
handleCreate asserts atoms is non-empty, so it crashes if we try to create one of them